Bottom: 46a346c7c65f204e67ea776c8e0358d3f03cf22c
Top:    603c3f85df2030265386aedeafe5ee15eed64686
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-05 19:01:33 +0100

Refresh of add-multi-clearance-drc-whilst

---

diff --git a/src/line.c b/src/line.c
index abad924..4dde0ca 100644
--- a/src/line.c
+++ b/src/line.c
@@ -657,10 +657,14 @@ EnforceLineDRC (void)
       if (TEST_FLAG (ALLDIRECTIONFLAG, PCB)) /* We don't have code to handle this case! */
         return;
 
+      /* Just run drc_lines to update clearances, without accepting any of its adjustment, when AUTODRCFLAG is not set */
       drc_lines (&rs, (PCB->Clipping == 2) != gui->shift_is_pressed ());
       return;
     }
 
+  if (TEST_FLAG (ALLDIRECTIONFLAG, PCB)) /* We don't have code to handle this case! */
+    return;
+
   /* first try starting straight */
   r1 = drc_lines (&rs, false);              /* XXX: This code doesn't cope well with all-direction lines (?) */
   /* then try starting at 45 */
